perl6/lib/Cutie.pm

```diff
@@ -1,3 +1,3 @@
 role Cutie {
-  has Bool $.accept-hugs is rw = False;
+  method accept-hugs { … }
```

Comment:
haha, yes! That's how you mark a place holder method that has to be defined by any class using the role. (You can also use `...`.) Perl6 has a bunch of unicode operators with optional ascii representations.